gramine_ratls_test.go

```diff
@@ -0,0 +1,263 @@
+package gramine_ratls
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// RA-TLS Certificate
+var certDer []byte
+var certPem []byte
+
+// RA-TLS measurements args
+var mrenclave []byte
+var mrsigner []byte
+var isv_prod_id []byte
+var isv_svn []byte
+
+func setup() {
```

Comment:
We should probably set the env var `RA_TLS_ALLOW_OUTDATED_TCB_INSECURE` here.

Reply:
Isn't that supposed to be insecure practice?